src/Symfony/Component/Notifier/Bridge/Plivo/Tests/PlivoTransportTest.php

```diff
@@ -0,0 +1,143 @@
+<?php
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Plivo\Tests;
+
+use Symfony\Component\HttpClient\MockHttpClient;
+use Symfony\Component\Notifier\Bridge\Plivo\PlivoTransport;
+use Symfony\Component\Notifier\Exception\InvalidArgumentException;
+use Symfony\Component\Notifier\Message\ChatMessage;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Test\TransportTestCase;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+use Symfony\Contracts\HttpClient\ResponseInterface;
+
+final class PlivoTransportTest extends TransportTestCase
+{
+    public function createTransport(
+        HttpClientInterface $client = null,
+        string $from = 'from'
+    ): PlivoTransport {
```

Comment:
One line

Reply:
Can't you guys run an auto-formatter on all PR code, instead of making the contribution process a frigging exercise in torture?

Reply:
Many things can be done via automated stuff, but we allow multiline arguments only on constructors, not on methods AFAIK.

You can check abbot and run the patch locally:
https://fabbot.io/report/symfony/symfony/48394/eeb2efe465020a056d4d14da65df323c55f0135e